.gitignore

```diff
@@ -32,4 +32,10 @@ src/*.egg-info
 # Ignore ESDL files created by workflows
 *_GrowOptimized.esdl 
 *_GrowOptimized_esdl_string.esdl 
-*_Simulation.esdl
\ No newline at end of file
+*_Simulation.esdl
+
+# Virtual environment files
+.venv/*
+
+# Documentation folder
+docs/*
```

Comment:
I added this because after compiling the documentation my git is now full of uncommited html files that were generated, that's why I added this to the gitignore file. What do you think is best? Leave this line here or should I delete my local html files?

Reply:
For me all the html files appear in docs/_build so I don't look at that folder. We shouldn't exclude the entire docs folder as we also write our documentation in there. You could potentially add docs/_build

Reply:
In the new commit instead of adding docs, I added docs/_build and docs/build_kvr_ to the gitignore file.